typing/layouts.ml

```diff
@@ -84,6 +84,22 @@ module Sort = struct
   and equate s1 s2 = match s1 with
     | Const c1 -> equate_sort_const s2 c1
     | Var v1 -> equate_var v1 s2
+
+  module Debug_printers = struct
+    open Format
+
+    let rec t ppf = function
+      | Var v   -> fprintf ppf "Var %a" var v
+      | Const c -> fprintf ppf (match c with
```

Comment:
I would turn that match into a `string_of_const` function elsewhere I think.

Reply:
I don't think that the debug printer I added will be problematic for Antal's change. It follows the format of other printers we list in `debug_printers.ml`.

Reply:
I've found that leaving them as printers is more convenient for consistency, but I'll make it a separate `let const ppf = ...` function above

Reply:
Oh heh and in doing that I fixed a small printing bug :-)